src/devdb/components/complete_quarters_report.py

```diff
@@ -0,0 +1,32 @@
+import streamlit as st
+import pandas as pd
+import plotly.express as px
+from src.constants import COLOR_SCHEME
+
+
+class CompleteQuartersReport:
+    def __init__(self, data):
+        self.complete_quarters = data["qaqc_quarter_check"]
+
+    def __call__(self):
+        st.subheader("Complete Quarter Distribution")
+        st.markdown(
+            """
+            This graph shows the distribution of records by complete quarter that were updated since the previous version.
```

Comment:
I tried splitting this first sentence up into two sentences 
> This graph shows the distribution of records by the quater they were completed. 
            Only records updated since the previous version are shown.
This reads a little better to me